### ISSUE TITLE
In game problem

### ISSUE THREAD

**Issue by haizor369** [L2J/L2J_Server#100]
Greetings, 
I am experiencing a problem with the filed i downloaded from the download section..
I downloaded the core master build and the master data and also the geodata
First of all when i put the geodata files inside the geodata folder i get an orrer about the memory usage
Secondly  the most important problem i face is the following:
When i log in the game and 5 - 10 minutes pass my character cannot move or target anything or wear items or use  skills/ admin commands.. Even if i use unstuck the same things continues to happen , even if i restart the game  the same things continue ... the problem is temporarly solved only if i exit the game and open a new window but in the next 5-10 minutes it happens again... I get no errors in the console when this happens.....

**Comment by Sacrifice78** [comment]
Try to do this https://github.com/L2J/L2J_Server/wiki/Geodata

**Comment by haizor369** [comment]
what about the second part... when i do not include the geodata the console runs smooth but i face the other problem i mentioned

**Comment by Sacrifice78** [comment]
Database config... locate my.ini file and config properly

**Comment by haizor369** [comment]
i also change the max memory to 3g but i still get the geodata bug...

**Comment by Sacrifice78** [comment]
Nots geodata config... sufficient with 3Gb. I mean database config... locate my.ini file (mysql config file) and config properly.

**Comment by haizor369** [comment]
oh and does the character problem i get relate to this also?

**Comment by haizor369** [comment]
i don't know what to change in that file help me please...

**Comment by Sacrifice78** [comment]
If you plan to run a server, you must have sufficient knowledgemens about MySQL, because if not, when enter 10 players... you have lag for not config database engine according with how do you want to run your server. Thats a personal setting.

**Comment by Sacrifice78** [comment]
Thats the difference between good or bad owners

**Comment by haizor369** [comment]
I wanna learn is there a way for me to learn through  here?  Or just give
me a quick fix for now i wanna test things on this server not mske it online
Στις 1 Οκτ 2015 2:11 μ.μ., ο χρήστης "Sacrifice" <notifications@github.com>
έγραψε:

**Comment by Sacrifice78** [comment]
you've opened an issue when it is your own problem, not L2J problem. You must not. @ Zoey76 can correct me if I'm wrong.

**Comment by haizor369** [comment]
Yes i am new user of this forum i did not know that i  should not post in
this section
Στις 1 Οκτ 2015 2:18 μ.μ., ο χρήστης "Sacrifice" <notifications@github.com>
έγραψε:

**Comment by Sacrifice78** [comment]
This seccion are for report issues that have to do with project not for ask for support on things that dont know.

**Comment by Sacrifice78** [comment]
Leave.post on forum not here

**Comment by haizor369** [comment]
But why would  mysql. Not allow me to. Use the character after an amount
of time...
Στις 1 Οκτ 2015 2:19 μ.μ., ο χρήστης "Sacrifice" <notifications@github.com>
έγραψε:

**Comment by Sacrifice78** [comment]
I am trying to tell you that you have to increase the number of concurrent connections in the my.ini file. the default is 150 if you installed well. If you installed MySQL developer I think the function sets the value of concurrent connections to 10.

I have not answered before that was in the hospital.

**Comment by haizor369** [comment]
So the problem that causes me to freeze after 5 minutes is due to this?
How much  should  i increase  it
Στις 1 Οκτ 2015 8:28 μ.μ., ο χρήστης "Sacrifice" <notifications@github.com>
έγραψε:

**Comment by Sacrifice78** [comment]
Probably. Are you running server with geodata included on 64 bits operating system and with a minimum of... 4096 Mb of RAM? (3072Mb for java and 1024 Mb for rest of operating system process)?.

**Comment by Sacrifice78** [comment]
Database engine (MySQL) is a handicap if are wrong configured too. I remember that 1000 concurrent connections on MySQL consume ~900 Mb of ram by MySQL process.

**Comment by Sacrifice78** [comment]
Ideal setting for a good server are that all operating server processes + memory for loginserver + memory for gameserver + memory for MySQL <= 75% of amount physical memory installed on server.

**Comment by haizor369** [comment]
But the problem i am having  happens even  if geodata is disabled...
Στις 1 Οκτ 2015 11:07 μ.μ., ο χρήστης "Sacrifice" <notifications@github.com>
έγραψε:

**Comment by Zoey76** [comment]
Post your system specs (OS, total ram, MySQL version, processor, etc).

**Comment by haizor369** [comment]
Windows 8.1 x64,  mysql 5.1,  installed 4gb Ram,  intel core i5-4200U
1.60-2.30GHz this  machine is only for personal use i wanna experiment  on
this server and learn..  The main problem i face as i already  told you is
that after  a period of time my character  can't  do anything rather exit
the game
Στις 3 Οκτ 2015 1:24 π.μ., ο χρήστης "Zoey76" <notifications@github.com>
έγραψε:
